tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -410,6 +410,16 @@ ecmp/inner_hashing/test_wr_inner_hashing_lag.py:
       - "asic_type not in ['mellanox']"
       - "'dualtor' in topo_name"
 
+ecmp/test_ecmp_sai_value.py:
+  skip:
+    reason: "Only support Broadcom T1/T0 topology with 20230531.11 or later image"
+    conditions_logical_operator: or
+    conditions:
+      - "topo_type not in ['t1', 't0']"
+      - "asic_type not in ['broadcom']"
+      - "release in ['201911', '202012', '202205', '202211']"
+      - "release in ['202305'] and build_version.split('.')[1].isdigit() and int(build_version.split('.')[1]) <= 12"
```

Comment:
hi @ZhaohuiS with this condition, if the image is 20230531.12, would this be skipped?